packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/OptOutModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { ReactNode, useState } from 'react';
+import styled from 'styled-components';
+import { Button, Image } from '@trezor/components';
+import { Translation, CheckItem, Modal } from 'src/components/suite';
+import { TranslationKey } from '@suite-common/intl-types';
+
+const CheckboxWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 32px;
```

Comment:
Nothing applied except for margin. Please use `spacingsPx`.

Reply:
fixed